randovania/games/samus_returns/logic_database/Area 5 - Exterior.txt

```diff
@@ -398,12 +398,9 @@ Extra - asset_id: collision_camera_008
   * Extra - actor_name: LE_Item_003
   * Extra - actor_type: item_senergytank
   > Door to Tower
-      Any of the following:
-          Lay Bomb
-          Lightning Armor and Morph Ball
-          All of the following:
-              Lay Power Bomb
-              Baby Metroid or Power Bomb ≥ 2
+      All of the following:
+          Morph Ball
+          Lightning Armor or Bomb or Power Bomb ≥ 2
```

Comment:
Was the removal of "Baby Metroid" intentional?

Reply:
The Baby is still a valid path to get to the item. All the Baby did on the return path was save a power pomb,bomb, which is kinda unnecessary.